_posts/2017-02-27-protocol-buffers.md

```diff
@@ -36,37 +36,32 @@ Bazel's proto rules implicitly depend on the
 distribution (described below, in "Implicit Dependencies and Proto Toolchains").
 The following satisfies these dependencies:
 
-> TIP: This is a shortened version of
-> [https://github.com/cgrushko/proto_library/blob/master/WORKSPACE](https://github.com/cgrushko/proto_library/blob/master/WORKSPACE)
+> TIP: Clone https://github.com/cgrushko/proto_library/ to try protobufs in Bazel now.
 
 ```python
-# proto_library rules implicitly depend on @com_google_protobuf//:protoc,
-# which is the proto-compiler.
+# proto_library, cc_proto_library, and java_proto_library rules implicitly
```

Comment:
Surround the rule and target names in backticks?

Reply:
These are inside a comment inside a code block - is it useful? they won't look different, and the backticks will be in the final result.

Reply:
I missed it, sorry.